fe/src/project/Seller.Portal/areas/Clients/components/ClientForm/ClientForm.js

```diff
@@ -62,6 +61,7 @@ function ClientForm(props) {
                 return response.json().then(jsonResponse => {
                     if (response.ok) {
                         setFieldValue({ name: "id", value: jsonResponse.id });
+                        setSaved(true)
```

Comment:
**question:** do we need it at all? Can't we detect whether the client is saved based on the id field?

Reply:
Yes